lir/registry.py

```diff
@@ -32,9 +35,11 @@ def _get_attribute_by_name(name: str) -> Any:
             else:
                 attr = __import__('.'.join(parts))
 
+            logging.debug(f'successfully imported: {name}. Found {attr}')
```

Comment:
Deze zou ik grammaticaal uitlijnen met regel 42 (het patroon dat je verwacht is enkele keren regel 42 en dan deze)

```suggestion
            LOG.debug(f'import successful: {name}. Found {attr}')
```